Update CRX known issues page (#6242)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/migrating/known-issues/index.md

{% endAside %}

> Recently, we announced changes to the Manifest V2 deprecation timeline, and while we remain firmly committed to Manifest V3 we acknowledge there is more work to do on our part including: 

Reviewer: ```suggestion
Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including: 
```

> Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including: 

Reviewer: ```suggestion
Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part.
```

Optional, but the "including" followed by bullets for "we will do <x>" sounds a little awkward.

*   Before announcing a new timeline for deprecation, we will finish addressing prioritized platform gaps and closing critical bugs that are documented on this page.

> *   We will give developers time to build, guaranteeing at least 6 months between a timeline announcement and any experiments deprecating MV2 features.

Reviewer: ```suggestion
*   We will give developers time to build, guaranteeing at least 6 months between a timeline announcement and any experiments removing support for MV2.
```

I feel like this is a bit more accurate (we may _separately_ deprecate features as part of normal platform maintenance)

## Closing the Platform Gap

We are committed to closing the following gaps before announcing a new MV2 deprecation timeline:

1. **User Script support:** Allow registering content scripts with arbitrary code by adding new functionality to the scripting API (see [our proposal](https://github.com/w3c/webextensions/blob/main/proposals/user-scripts-api.md) for more detail).
1. Further **Service Worker lifecycle enhancement:** 

>     * Additional strong Service Worker keepalives for operations taking longer than 5 minutes.

Reviewer: ```suggestion
    * Additional strong Service Worker keepalives for certain operations taking longer than 5 minutes.
```

>     * Additional strong Service Worker keepalives for operations taking longer than 5 minutes.

Reviewer: ```suggestion
    * Additional strong Service Worker keepalives for certain operations taking longer than 5 minutes.
```

    * Extend service worker lifetimes while there are active WebSocket connections.
1. **Increase the number of static and enabled rulesets** for Declarative Net Request (DNR).
1. Extend **[Offscreen document](/docs/extensions/reference/offscreen/) functionality** to support more reasons for using an Offscreen document.

> 1. Improving **support for the `chrome.tabCapture` API**:

Reviewer: ```suggestion
1. Improving **support for the [`chrome.tabCapture`](https://developer.chrome.com/docs/extensions/reference/tabCapture/) API**:
```

    * Support calling `getMediaStreamId` from a service worker.
    * Support obtaining a MediaStream from a stream ID in an offscreen document.

> 1. **Support File Handling API on ChromeOS** as a replacement for `chrome.fileBrowserHandler`.

Reviewer: ```suggestion
1. **Support File Handling API on ChromeOS** as a replacement for [`chrome.fileBrowserHandler`](https://developer.chrome.com/docs/extensions/reference/fileBrowserHandler/).
```
